src/modules/oh-indicators-list/store/slice.ts

```diff
@@ -41,17 +48,22 @@ export const indicatorsSlice = createSlice<IState, any>({
       )
       .addCase(
         getAllItems.rejected,
-        (state, action: PayloadAction<any, any>) => {
-          state.error = (action as any).error.message;
+        (state, action) => {
+          state.error = action.error.message;
```

Comment:
👍
so adding 3rd param to getAllItems generic fixed it

Reply:
yes :)